src/components/organisms/EntityList/EntityCreationModal/TestCreationModalContent.tsx

```diff
@@ -40,27 +40,26 @@ const TestCreationModalContent: React.FC = () => {
 
     if (!selectedExecutor) {
       setHintConfig(defaultHintConfig);
-      return;
+    } else {
+      if (selectedExecutor.executor?.executorType === 'container') {
```

Comment:
This way it will render the component even when there is no selected executor. Is that expected?

Reply:
Once we chose some `executorType` we can't nullify it. So basically I adjusted this logic without having an early return pattern. The business logic was not changed 👌